### ISSUE TITLE
Issue with Season pack search

### ISSUE THREAD

**Issue by pedro2d10** [SickRage/SickRage#3113]
Hello, 

I think i found an issue with the season pack search. Indeed, for the show "test", the "SEARCHQUEUE-BACKLOG" only search for the show like "test%20S01". Otherwise, i know that the show exist in the torrent provider as "test%20season01" (or saison01 for the french guy ;))

Is it possible to do something for me?
Thanks

### Before submitting your issue:

Enable debug logging in SickRage settings, reproduce the error (be sure to disable after the bug is fixed)

Branch/Commit: master
OS: Linux CentOs

**Comment by SickRage2** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [ ] Post debug logs, either inline (for smaller logs) or using gist

Please make sure you also read [how to create an issue](https://github.com/SickRage/sickrage-issues/blob/master/README.md) and followed all of the steps. 

The title should describe your issue. Having "SR not working" or "I get this bug" for 100 issues, isn't really helpful. We will close issues if there isn't enough information.

Sometimes the devs may seem like grunts and respond with short answers. This isn't (always) because the dev hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let us know, and this bot may get updated to automatically answer you.

Thanks!

**Comment by sharkykh** [comment]
@miigotu 
Is it a simple addition?
https://github.com/SickRage/SickRage/blob/master/sickrage/providers/GenericProvider.py#L437

**Comment by miigotu** [comment]
search_string['Season'] is a list of strings that will be searched, so yeah basically its a simple addition, but you have to add the extra strings only in that else.

_get_episode_search_strings does it for anime how they want it on seasons.

**Comment by sharkykh** [comment]
Yeah, that's what I though,
but now your last sentence made me confused.
I think you meant **_get_season_search_strings** does it for anime how the user wants, right?

**Comment by pedro2d10** [comment]
Yeah, i just read the code from GenericProvider. Indeed, when the show is an anime it do the trick. 
But nom my problem is for french provider: "Saison" instead of "Season".
The only way the trik will do the job for me, is to add in parameter, something to tell SR the string to search for season pack.
Thanks both of you the support ;)

**Comment by miigotu** [comment]
ideally this would be a per-show option to set the season string type.
And no I meant to look at how anime is done in `_get_episode_search_strings` because it adds a second string for anime shoes there, with search_string_alt or w/e

**Comment by h3llrais3r** [comment]
I would not start with implementing language dependent queries... What we could do is adding a fallback to search for 'season' next to the default 'S0X' for non anime seasons. 
For anime we only search for 'season', even without the number, to have as much as matches as possible.
The ideal way is like mentioned by miigotu, but that's a feature request.

**Comment by miigotu** [comment]
We could set this per provider fairly easy. Most providers do cater to a specific style/language.  Would that suffice?
